### PR TITLE
chore(.github): add workflow for JA full translation by gcp

### PR DESCRIPTION
**Pull request by Yuiham**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add full translation workflow of Japanese for TiDB LTS version.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

File: .github/workflows/ja-full-translation-google.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    steps:
+      - uses: actions/checkout@v3
+        name: Download translator repo
+        with:
+          repository: "shczhen/markdown-translator"
+          path: "markdown-translator"
+      - uses: actions/checkout@v3
+        name: Download specified branch of docs repo
+        with:
+          ref: "release-7.5"
```
[the remark this answers is not in the document]
Comment (Yuiham) [reply .github/workflows/ja-full-translation-google.yaml:18]: Let's finish `v7.5` translation first, and then I will parameterize the LTS `version`.

File: .github/workflows/ja-full-translation-google.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
+name: JA Full Translation (Google version)
+
```
[the remark this answers is not in the document]
Comment (Yuiham) [reply .github/workflows/ja-full-translation-google.yaml:2]: Fixed.

**Review by shhdgit — APPROVED** [review]
lgtm.

**Review by ran-huang — APPROVED** [review]

**Pull request merged**